packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -108,7 +110,13 @@ export const OutboundShipmentListViewComponent: FC = () => {
   const columns = useColumns<Invoice>(
     [
       getNameAndColorColumn(onColorUpdate),
-      'status',
+      [
+        'status',
+        {
+          formatter: (status, { t }) =>
+            t(getStatusTranslation(status as OutboundShipmentStatus)),
+        },
+      ],
```

Comment:
not quite as nice as just passing a string.. 😁  but I think it still ends up being quite nice..

Reply:
maybe not as simple as just a string.. still very neat tho